src/client/index.js

```diff
@@ -60,15 +60,9 @@ if (typeof window !== 'undefined' && !__NEXTAUTH._eventListenersAdded) {
 }
 
 // Context to store session data globally
+/** @type {import("types/internals/client").SessionContext} */
 const SessionContext = createContext()
 
-/**
- * React Hook that gives you access
- * to the logged in user's session data.
- *
- * [Documentation](https://next-auth.js.org/getting-started/client#usesession)
- * @type {import(".").UseSession}
- */
 export function useSession (session) {
```

Comment:
I *THINK* this is not necessary, since we have the `.d.ts` file right next to it in the root. JSDoc types defined in the `.js` will be overridden with the `.d.ts` ones.

Would have been easier if we had `.ts` files, but this is a separation we chose for now.